test/infrastructure/vcsim/config/crd/bases/vcsim.infrastructure.cluster.x-k8s.io_vmoperatordependencies.yaml

```diff
@@ -96,9 +96,9 @@ spec:
                   datacenter:
                     description: supervisor is based on a single vCenter cluster
                     type: string
-                  folder:
+                  distributedPortGreoupName:
                     type: string
-                  networkName:
+                  folder:
```

Comment:
What happened to networkName?

(guess: this file was outdated before this PR and we don't run the generate target in our verify target for CI?)

Let's follow-up

Reply:
Network name was dropped, we are not using it anymore for the vmOperator

Reply:
Sorry my bad. All good. Not sure how I missed that we actually dropped the field. I guess I was looking at the diff with 2 fields and somehow got confused :)